src/scene/geometry/geometry.js

```diff
@@ -91,6 +98,15 @@ class Geometry {
         Debug.assert(this.indices, 'Geometry must have indices set');
         this.tangents = calculateTangents(this.positions, this.normals, this.uvs, this.indices);
     }
+
+    /**
+     * Generates triangle information from the positions and triangle indices.
+     */
+    calculateTris() {
```

Comment:
I would suggest, for now, removing this function and have `TriData` call `calculateTris`. And also for now, just keep `calculateTris` private/not exported. We can expose it later should we wish, but let's be conservative for now.